### ISSUE TITLE
Главное меню (п.11): обновление вида

### ISSUE THREAD

**Issue by nicothin** [nicothin/nesia#25]
@vladimir075 @nadezdafrolova 

Макет: https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3141%3A3277

Задача: для мобилки перенос вправо, смена структуры (появляется вложенность).

Вопросы:

1. Какова структура пунктов меню для настольной и мобильной версий, для авторизованного пользователя?
2. Как выглядит вложенность меню для настольной версии?

**Comment by vladimir075** [comment]
в Мобилке добавляется 1) вложенность разделов стран и переключалки языков 2) внизу кнопка List your property fo free.
НО! очень важно сохранить и старую текущую структуру меню, Надежда здесь ее игнорировала
Это 
- аватар с приветствием
- линии логического разграничения пунктов
- зеленый значек входящих сообщений
Если для верстки необходимо перечислить пункты и их последовательность, могу это сделать
<img width="679" alt="Screenshot 2022-05-23 at 1 11 51 PM" src="https://user-images.githubusercontent.com/17883276/169797729-35b3ef38-a4e5-4870-8afe-d81ba7b6b6a2.png">

**Comment by nicothin** [comment]
https://nicothin.pro/nesia/countries.html

по факту, добавился пункт меню со ссылкой на страны и дроп с языками.
судя по макету, пункт "Страны" именно ссылка на отдельную страницу
крестик закрытия я оставил на том же месте, где бургер, ибо плохо, когда интерактивный элемент перемещается.

**Comment by vladimir075** [comment]
при клике на ссылку выходит страница 404, не могу посмотреть

**Comment by nicothin** [comment]
> при клике на ссылку выходит страница 404, не могу посмотреть

Удивительно.
Опубликовал полную копию страницы с другим именем: https://nicothin.pro/nesia/countries-copy.html

**Comment by vladimir075** [comment]
а где посмотреть не "корневую". Там ведь язык у нас туда еще перенесся с подвала. Или он просто делается Олегом по примеру этой вкладки Countries?

**Comment by nicothin** [comment]
> а где посмотреть не "корневую". Там ведь язык у нас туда еще перенесся с подвала. Или он просто делается Олегом по примеру этой вкладки Countries?

Да, "корневая" отличается только отсутствием хлебных крошек над заголовком, насколько я могу судить.

Через пару минут развернутся эти 2 ссылки на замену всех предыдущих. Одна с крошками, другая без.

https://nicothin.pro/nesia/countries-1.html
https://nicothin.pro/nesia/countries-2.html

**Comment by vladimir075** [comment]
а добавление туда смены языка интерфейса, просто по этому аналогу делается?

**Comment by nicothin** [comment]
> а добавление туда смены языка интерфейса, просто по этому аналогу делается?

В смысле, просто перевод на другие языки этого текста? Да, пока на проекте нет языков с направлением чтения, отличным от "слева направо".

**Comment by vladimir075** [comment]
Я не про то, мы значек смены языка перенесли из подвала в меню

**Comment by nicothin** [comment]
Пока проблемы не вижу. Возникнет - уверен, мы её решим.
